### ISSUE TITLE
Should developer documentation  mention that redflags-database.sql is needed

### ISSUE THREAD

**Issue by KrzysztofMadejski** [petabyte-research/redflags#18]
Should the docs here http://docs.redflags.eu/developer/first/ mention that redflags-database.sql is needed for engine to work?

Or is the `redflags-auto.sh` checks if the database exists and run the script if not?

I've run the script and it seems going without initiating the db. On the other hand i see that db has no tables. So I guess it will fail at later point, will it?

**Comment by KrzysztofMadejski** [comment]
Should it mention also `redflags-helper-tables.sql`?  Should I run both? If yes, then, shouldn't these scripts be merged?

**Comment by dzierzy** [comment]
redflags-database.sql script should be described in docs. 

redflags-helper-tables.sql is covered by sh script, it's being executed after engine finish processing procurements

**Comment by juzraai** [comment]
As I wrote it in the ToC, I constructed the documentation assuming developers will read through the first two sections (user, maintainer) too before reading developer docs. :)

Maintainer documentation tells every step required to set up the database, including both scripts: http://docs.redflags.eu/maintainer/database/

> shouldn't these scripts be merged?

Nope. :)
- `redflags-database.sql` is required only once when you set up the database.
- `redflags-helper-tables` is required after every run of the engine.
